#165413096 Integrate with coveralls

AJAkimana opened the pull request:
Description
=======
This is to ensure we are properly informed just how well our code base is tested which in turn provides quality assurance for the product

Type of change
=======
N/A

How has it been tested
=======
N/A

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#165413096](https://www.pivotaltracker.com/n/projects/2326984/stories/165413096)

Commit:
chore(task): Integrate with coveralls
[finishes #165413096]

macphilips commented on .babelrc:
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-env"]
+}
```
Add new line

AJAkimana replied:
> Add new line

:all-done

Commit:
chore(task): Add coverage badge in README file

[#165413096]

macphilips commented on package.json:
```diff
@@ -6,11 +6,19 @@
     "lint": "eslint --ignore-path .gitignore .",
-    "lint:fix": "npm run lint --fix"
+    "lint:fix": "npm run lint --fix",
+    "migrate": "",
```
Remove this line. it's of no use

AJAkimana replied:
> Remove this line. it's of no use
I am On it

AJAkimana replied:
> Remove this line. it's of no use
it's done

Niyitangasam reviewed (changes requested):
I am not able to run this locally.
![image](https://user-images.githubusercontent.com/12936280/56849275-d1a2ea00-68f2-11e9-8c9f-c810240e61d7.png)

Commit:
chore(task): Delete unused script

- Remove npm run migrate
- Add .env in gitignore
- Remove package-lock.json from gitignore
- Remove second .env in gitignore
[#165413096]

ericnyirimana reviewed (approved):
It works great, good work

macphilips commented on .gitignore:
```diff
@@ -34,9 +34,16 @@ node_modules
+
+yarn.lock
+
+package-lock.json
```
We already have package-lock.json defined in this file kindly remove this

macphilips commented on .gitignore:
```diff
@@ -34,9 +34,16 @@ node_modules
 .node_repl_history
 
+# Environnement
+.env
```
We already have .env defined in this file kindly remove this
